scripts/tags/exturl.js

```diff
@@ -0,0 +1,57 @@
+/* global hexo */
+// Usage: {% exturl text url "title" %}
+// Alias: {% extlink text url "title" %}
+
+'use strict';
+
+var util = require('hexo-util');
```

Comment:
Ok, i just reproducte base install with it. In main `node_modules` dir there is no `hexo-util` module.
I think need to use some global var of hexo `node_modules` but can't find it for now.
Alose, some bug was find, i try to fix it soon.